setup_scripts/install_tools.sh

```diff
@@ -110,7 +94,7 @@ cd $DIR
 # Setup Snowbots Udev Rules  #
 ##############################
 cd $DIR
-./src/firmware/setup_udev_rules.sh
+./../src/firmware/setup_udev_rules.sh
```

Comment:
I think we should move all the script files out of src, maybe place setup_udev_rules.sh into this $DIR. Another thought is the udev setup for phidget there are two ways to look at it. 1. Move all the udev rules into one file. 2. Move all the udev rules to their respective scripts. (i.e If udev setup_udev_rules.sh is for arduino, then move udev setup into the arduino install script)

Reply:
I like the idea of having a unified udev rules file and having that file and the script to install it under `setup_scripts`. I've moved the `setup_udev_rules.sh` and `10-snowbots.rules` into `setup_scripts`.